### ISSUE TITLE
conda(-forge) package?

### ISSUE THREAD

**Issue by hmaarrfk** [datoviz/datoviz#59]
How would you feel about making a conda-forge package?

Do you feel like datoviz is ready for a more general audiance?

If not, I can just publish it for "myself" on "my own channel".

See work being done:
https://github.com/conda-forge/staged-recipes/pull/23164

**Comment by rossant** [comment]
The current version is still highly experimental and will be superseded by a new version in the next months. I won't have the bandwidth to do much user support on the current version. That being said, it will be definitely helpful to have an easily-installable version with conda for early adopters and colleagues in my own organization.

If you publish it on your own channel, how would users do it to install it in practice?

**Comment by hmaarrfk** [comment]
Yes. That would work.

But on the `mark.harfouche` channel, I just delete things whenever I want.

**Comment by djhoese** [comment]
Is the build process for the current version and this future version relatively similar @rossant? In that case, if @hmaarrfk is willing to spend the time trying to move the build process into a conda-forge recipe maybe that is time well spent. Updating the recipe version when the build process is the same/similar would be easy. It might also be good to figure out what dependency issues arise from trying to build with conda-forge packages and to identify any issues the conda-forge admins have with the package.

Just a thought.

**Comment by rossant** [comment]
Yes, I agree. I think the build processes should be relatively similar.

**Comment by hmaarrfk** [comment]
from what I gather here, we should;

1. wait a few months for your new release.
2. put what we can on conda-forge

Honestly, I'm having trouble getting mac + windows to work on conda-forge. so help on the windows dependencies, and maybe underestanding why a segfault is happening on mac would be helpful in the meantime.

**Comment by rossant** [comment]
The latest version includes prebuilt pip wheels, so a conda package may not be necessary for now.

**Comment by hmaarrfk** [comment]
> necessary

I think conda vs pip have different advantages. 

Has the development situation of dataviz significantly changed  since mid 2023? 

Is it time for me to revisit this?
